parser_elixir/README.md

```diff
@@ -0,0 +1,21 @@
+# ParserElixir
+
+**TODO: Add description**
+
+## Installation
+
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed
+by adding `parser_elixir` to your list of dependencies in `mix.exs`:
```

Comment:
is that relevant?

Reply:
this is the standard mix scaffolded README I guess it is only noise in this context, I'll remove it too.